qiskit/execute_function.py

```diff
@@ -148,10 +148,12 @@ def execute(
         optimization_level (int): How much optimization to perform on the circuits.
             Higher levels generate more optimized circuits,
             at the expense of longer transpilation time.
-            #. No optimization
-            #. Light optimization
-            #. Heavy optimization
-            #. Highest optimization
+
+                * No optimization
+                * Light optimization
+                * Heavy optimization
+                * Highest optimization
+
```

Comment:
`#.` makes a numbered list, and I think this list is logically meant to look like "0. No optimisation; 1. Light optimisation; ...".  If so, I think this was intended:
```rst
... at the expend of longer transpilation time.

0. No optimization
#. Light optimization
#. Heavy optimization
#. Highest optimization
```

(`#.` autonumbers in rst)

Reply:
oops.. yes, my intentation was to unified it with the previous style: 
```
              * 0: no optimization
              * 1: light optimization
              * 2: heavy optimization
              * 3: even heavier optimization
```

done in 915c7e96b59b0721344d0c8f7bee61df1211d099